### DISCUSSION TITLE
Custom entry page instead of index.html for static website

### DISCUSSION THREAD

**Discussion by tsathis** [vercel/vercel#5303] [Help] [answered]
I know this is a simple question but I couldn't get how to do it. 

Assume that I have a static website like,
```
├───some_files
├───hi.html
└───index.html
```
Normally when I deployed Vercel would use `index.html` as the entry page by default. How can I configure vercel.json such that it uses the `hi.html` as the entry page?

**Comment by styfle** [comment, marked as answer]
Yes, you can add `vercel.json` with [rewrites](https://vercel.com/docs/configuration#project/rewrites) property.

```json
{
  "rewrites": [
    { "source": "/", "destination": "/hi.html" }
  ]
}
```

**Reply by tsathis** [reply]
Actually it worked, but only after deleting `index.html`. It seems like when there is an `index.html`, it use that file. Is it?

**Reply by styfle** [reply]
Oh yes, my mistake. When you use `rewrites` priority is given to the filesystem. You can use `routes` if you have a index.html file but want to override it.

```json
{
  "routes": [
    { "src": "/", "dest": "/hi.html" }
  ]
}
```

**Reply by tsathis** [reply]
Yeah it worked, thank you!

**Reply by firdavsyusupov** [reply]
Hello, if I want to set /index.html#contact to /contact
How to edit this?

**Reply by leerob** [reply]
@firdavsyusupov Here you go -> https://sdk.vercel.ai/s/kfsW2N57XwVcvrPduFH4j
